chain/gen/genesis/miners.go

```diff
@@ -472,9 +473,9 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sroot cid.Cid
 		return cid.Undef, xerrors.Errorf("TotalRawBytePower (%s) doesn't match previously calculated rawPow (%s)", pc.RawBytePower, rawPow)
 	}
 
-	if !pc.QualityAdjPower.Equals(qaPow) {
-		return cid.Undef, xerrors.Errorf("QualityAdjPower (%s) doesn't match previously calculated qaPow (%s)", pc.QualityAdjPower, qaPow)
-	}
+	//if !pc.QualityAdjPower.Equals(qaPow) {
+	//return cid.Undef, xerrors.Errorf("QualityAdjPower (%s) doesn't match previously calculated qaPow (%s)", pc.QualityAdjPower, qaPow)
```

Comment:
`pc.QualityAdjPower` then is equal to `824572800`, whereas `qaPow` is calculated to be `838860800` (we are using 8MiB sectors for the lotus-soup testplans).